hiveeyes/sensor_ds18x20.py

```diff
@@ -26,7 +31,10 @@ def start(self):
 
         # Initialize the DS18x20 hardware driver.
         try:
-            from onewire.onewire import DS18X20
+            if platform_info.vendor == MicroPythonPlatform.Vanilla:
+                from ds18x20 import DS18X20
```

Comment:
Will this module really be available on all Genuine MicroPython platforms out of the box?

Reply:
Yes
https://docs.micropython.org/en/latest/esp32/quickref.html#onewire-driver

Reply:
Thanks. We will see if this is also available on other MCU architectures beyond ESP32.